packages/vorbis_rs/Cargo.toml

```
@@ -33,6 +33,7 @@ getrandom = { version = "0.3.0", features = ["std"], optional = true }
 [features]
 default = ["stream-serial-rng"]
 stream-serial-rng = ["dep:getrandom"]
+wasm_js = ["getrandom/wasm_js"]
```

Since the `getrandom` dependency is not be included when the `stream-serial-rng` feature is disabled, which happens when disabling the default features for `vorbis_rs`, it's best to enable the `wasm_js` feature only if `getrandom` is already a dependency, as explained in [the relevant Cargo documentation](https://doc.rust-lang.org/cargo/reference/features.html#dependency-features).

Additionally, since existing features are named using `kebab-case`, it would be more consistent to follow the same naming convention for new features.

```suggestion
wasm-js = ["getrandom?/wasm_js"]
```

---

good catch there, I'm kinda learning rust at the same time so this is helpful thanks